cytosnake/utils/cytosnake_setup.py

```diff
@@ -0,0 +1,120 @@
+"""
+Module: cytosnake_setup.py
+
+This modules sets up the current directory as a project directory. Project
+directories are dictated by the presence of `.cytosnake`, which contains 
+meta data for cytosnake to use. 
+
+"""
+import json
+import shutil
+
+from pathlib import Path
+from cytosnake.utils.cyto_paths import (
+    get_cytosnake_package_path,
+    get_config_fpaths,
+    get_workflow_fpaths,
+    get_project_dirpaths,
+)
+
+from cytosnake.common.errors import display_error
+
+
+def create_cytosnake_dir() -> None:
+    """Sets up a `.cytosnake` folder that contains meta information that helps
+    cytosnake's cli functionality. This will include configurations that
+    contains states, paths and settings.
+
+
+    Returns
+    -------
+    None
+        Generates a `.cytosnake` file in current directory
+
+    Raises:
+    FileExistsError
+        Raised by pathlib.Path if `.cytosnake' exists in cur
+    ."""
+
+    # create the `.cytosnake` folder in where `cytosnake init` mode was executed
+    # - if the current director has a `.cytosnake` FileExistError will be raised
+    cwd = Path().absolute()
+    project_folder_path = cwd / ".cytosnake"
+    try:
+        project_folder_path.mkdir(exist_ok=False)
+    except FileExistsError as e:
+        msg = "'.cytosnake' directory exists in current directory"
+        display_error(e, msg=msg)
+
+
+# TODO: This should be in the `file_utils` module, may trigger circular imports
```

Comment:
Is this PR out of scope to perform this TODO?

Reply:
Yeah I was getting some pesky circular import errors and I have not figured that out yet. Might be a seperate PR. This is a temporary solution.

Reply:
Issue?

Reply:
I agree. Created issue #28